integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/altinn/MessagePolling.java

```diff
@@ -70,10 +73,13 @@
     @Autowired
     ConversationRepository conversationRepository;
 
+    @Autowired
+    private NextBestQueue nextBestQueue;
+
     private ServiceRecord serviceRecord;
 
     @Scheduled(fixedRate = 15000)
-    public void checkForNewMessages() {
+    public void checkForNewMessages() throws IOException, MessageException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.io.IOException, no.difi.meldingsutveksling.noarkexchange.MessageException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1160)